error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -615,6 +615,58 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link Mono#flatMap(Function)} over more contrived alternatives. */
+  static final class MonoFlatMapIdentity<S, T> {
```

Comment:
We generally name the patterns after the `@AfterTemplate`. So this would be `MonoFlatMap`.

Reply:
I didn't get this pattern beforehand - notes :memo:

Reply:
No worries; we plan to automate this :smile: